CHANGELOG.md

```diff
@@ -6,7 +6,19 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.11...develop
+[Unreleased]: https://github.com/bottlerocket-os/bottlerocket-test-system/compare/v0.0.13...develop
+
+## [0.0.13] - 2024-06-06
+
+- cluster: add function to download eks-a bundle file ([#895])
```

Comment:
Ah, I missed this on my first read-through, but it seems like we should add an `### Added` header here.